### DISCUSSION TITLE
SimpleMenu (App+Games) not working

### DISCUSSION THREAD

**Discussion by sthamann** [eduardofilo/RG350_adam_image#137] [SimpleMenu topics]
Hi,
if i want to start "SimpleMenu" from the MainMenu the System is loading for 1 or 2 seconds and then returning to the main menu.
I have the latest version of adam-image installed and the external sd card is formated to FAT32 and without a name.
Any idea what the problem may be or where i could find a log file with additional information?
I don´t have any preview folders created yet (btw: is there a tool to automate this?)

**Comment by eduardofilo** [comment]
Hi.
I'm not sure I understand you, sorry. With MainMenu you mean GMenu2X?
If so, how did you manage to disable SimpleMenu as the default? Did you use the `B` shortcut while booting?
If the problem is the SimpleMenu doesn't work, the problem is always related with a lack of sync between the last_state.sav file and the content in your EXT card. Try deleting that file to reset the status of SimpleMenu and begin with the EXT card empty. Then begin to add the contents little by little trying to open SimpleMenu in each step.
The previews folders are created automatically while system boot. You should see a `.previews` folder inside each roms directory. If you don't see that folder scaffolding, you have problems with your EXT SD card. Or it has label, or it is not formatted in FAT32 or it has more than a partition. Double check those topics.

**Comment by LianaVena** [comment]
Hi, I have the same issue on my `RG300X`. I have gone through the setup of internal and external SD card and when I press `A` to go to `SimpleMenu`, it goes through loading and then goes back to previous menu. 

I didn't use `B` shortcut while booting. I couldn't find `last_state.sav` on the SD card. I tried with empty external card and with some GB games added. I can see `.previews` on external card. 

Adding a video of the issue:
https://user-images.githubusercontent.com/88106055/148681908-7566dd90-673e-4c5a-8ce9-712fc044ba47.mp4

**Reply by eduardofilo** [reply]
Hi. I have seen your video. Thanks for doing it by the way. It helps a lot to debug the problems.
You are trying to open SimpleMenu while you are already in SimpleMenu, which doesn't make sense.

Now that I see that this confusion can occur, it gives me the feeling that the same thing happened to @sthamann.
In that case, for both of you, take a look at FAQ number 10, especially the video at the end.
https://github.com/eduardofilo/RG350_adam_image#10-why-dont-appear-in-simplemenu-the-roms-i-just-loaded-onto-the-external-card

**Reply by LianaVena** [reply]
Hi, I have tried pressing B and it indeed just was already in that sub menu. On next start-up it remembers where I was and works well. Thanks!
